Warcraft-Guardians-of-Azeroth-2-Music-Submod.mod

```diff
@@ -4,4 +4,4 @@ tags={
 }
 name="Warcraft Guardians of Azeroth 2 Music Submod"
 supported_version="1.5.0.2"
-path="C:/Users/akozh/Documents/Paradox Interactive/Crusader Kings III/mod/Warcraft-Guardians-of-Azeroth-2-Music-Submod"
\ No newline at end of file
+path="/mod/Warcraft-Guardians-of-Azeroth-2-Music-Submod"
```

Comment:
The extra / at the start of the mod directory was causing problems for me. I had to delete it for GoA2 launcher to detect the mod properly.

I suggest changing this to `path="mod/Warcraft-Guardians-of-Azeroth-2-Music-Submod"`